GOVERNANCE.md

```diff
@@ -0,0 +1,68 @@
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* In general continue to be willing to spend at least 25% of ones development time on CRI-O (~1.25
+  business days per week).
+
+## Cutting a release
+
+* We cut releases in line with Kubernetes minor releases (1.x bump). We try to cut the release as quickly as possible after a Kubernetes release, but have been known to wait up to a month after the release, to allow for features to make it into a certain release.
+* For Patch releases (1.1.z bump), releases are cut intermittantly, when there are sufficient bug fixes backported to the branch. End-users can request releases be cut, and maintainers can choose to accept that request at their discretion.
+* Release notes are compiled by our [release-notes generator](https://github.com/cri-o/cri-o/blob/master/scripts/release-notes/release_notes.go).
+* A release can be cut with our [release script](https://github.com/cri-o/cri-o/blob/master/scripts/release/release.go).
+* Finally, create a [tagged release](https://github.com/cri-o/cri-o/releases). The release should  start with "v" and be followed by the version number. E.g., "v1.18.0". *This must match the corresponding Kubernetes release number.*
+
+## When does a maintainer lose maintainer status
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they may volunteer to be removed from the [OWNERS list](github.com/cri-o/cri-o/tree/master/OWNERS).
+
+In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+
+They may also opt to be come an [emeritus maintainer](#emeritus-maintainer)
+
+# Emeritus Maintainer
+
+An emeritus maintainer is a maintainer who used to spend a significant amount of time on the project, but now has refocused their efforts.
+
+The CRI-O community has the emeritus maintainer tier to recognize the impact and input early maintainers have had on the project, while also being realistic about their current committment level.
+
+There is no formal process for becoming an emeritus maintainer: a maintainer simply slows down on their contributions but does not remove themselves from the [OWNERS file](OWNERS.md).
+
+# Feature addition policy
+
+Features are added by simply adding a PR. The maintainers handle this the same as a bug fix PR, discussing whether the feature will be added in the PR page.
+
+# Conflict resolution and voting
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out between the persons involved. If a dispute cannot be decided independently, the maintainers can be called in to decide an issue. If the maintainers themselves cannot decide an issue, the issue will be resolved by voting. The voting process is a simple majority in which each maintainer receives one vote. Votes not received within the span of one week are treated as an abstention.
```

Comment:
Any tie breaker procedures?  If you list them, you'll never need them, if you don't....
Perhaps just "In the case of a tie vote, the issue will be considered to have not passed."